docs/lib/graphqlapi/fragments/flutter/advanced-workflows/10_example.md

```diff
@@ -0,0 +1,28 @@
+```dart
+try {
+    String graphQLDocument = '''mutation createTodo(\$input: CreateTodoInput!) {
+      createTodo(input: \$input) {
+        id
+        name
+        description
+      }
+    }''';
+
+    var operation = Amplify.API.mutate(
+        request: GraphQLRequest<String>(
+            document: graphQLDocument,
+            variables: {
+              "name": "my first todo",
+              "description": "todo description"
+            }
+        )
+    );
+
+    var response = await operation.response;
+    var data = response.data;
+    
+    print("Mutate SUCCESS");
+} catch(e) {
+    print("Mutate FAILED");
+}
```

Comment:
Is there a specific reason to capitalize `SUCCESS` and `FAILED`. They are also not following the same verb tense.

Reply:
These files are actually not being used. Removed.